vertexai/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "quickstart-js-storage",
+  "version": "1.0.0",
+  "description": "JavaScript quickstarts for Firebase Storage.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/firebase/quickstart-js.git"
+  },
+  "author": "",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/firebase/quickstart-js/issues"
+  },
+  "engines": {
+    "npm": ">=9.0.0 <10.0.0",
+    "node": ">=18.0.0 <=20.0.0"
+  },
+  "homepage": "https://github.com/firebase/quickstart-js#readme",
+  "devDependencies": {
+    "typescript": "^5.1.6",
+    "vite": "^4.4.9"
+  },
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "format": "prettier --write ."
+  },
+  "dependencies": {
+    "firebase": "10.12.0"
```

Comment:
nit, the other SDKs look like they use ^.

`"firebase": "^10.12.0"`

Reply:
Done.